src/openapi-generator.ts

```diff
@@ -868,8 +879,8 @@ export class OpenAPIGenerator {
   ): SchemaObject | ReferenceObject {
     return omitBy(
       {
-        ...initialData,
         ...this.buildSchemaMetadata(metadata),
+        ...initialData,
```

Comment:
Hmmm what caused this change? I think that the metadata should override anything coming as a base schema.